clarify build on docs content and move doc to dag section

mirnawong1 opened the pull request:
this pr sets to update the following items to clarify dbt explorer and dbt docs usage as there's been confusion per user feedback:

- updates artifacts page to mention dbt explorer, updates screenshots
- updates documentation page to mention explorer and clarifies which configs are avaialble for egacy docs.
- moves documentation page to 'build your dag' so it follows dbt workflows
- reworks 'build your docs in dbt cloud' and keeps it under 'collaborate. 
- renames 'explore dbt projects' to 'navigate dbt explorer' so the product/feature name is visible in docs. 
- updates the links and adds redirects

Resolves #4882
Resolves #5304 
[docs project](https://www.notion.so/dbtlabs/dbt-Explorer-and-legacy-docs-clarification-52b7a372082249e6a06b8cb5e7d1662c?pvs=4)

Commit:
rephrase

Commit:
add explorer info

Commit:
add img

Commit:
updates to build your docs

Commit:
add redirect and update links

Commit:
add links

Commit:
update screenshots

mirnawong1 committed:
Merge branch 'current' into mwong-update-build-docs

mirnawong1 commented on website/docs/docs/collaborate/explore-projects.md:
```diff
@@ -1,28 +1,33 @@
-    - There has been at least one successful job run in the deployment environment. Note that [CI jobs](/docs/deploy/ci-jobs) do not update dbt Explorer. 
-- You are on the dbt Explorer page. To do this, select **Explore** from the top navigation bar in dbt Cloud.
-
+- You have at least one successful job run in the deployment environment. Refer to [set up a documentation job](/docs/collaborate/set-up-doc-job) for details on how set up a job run. Note that [CI jobs](/docs/deploy/ci-jobs) do not update dbt Explorer.
```
confirm for tech accuracy

dave-connors-3 replied:
this looks right to me

mirnawong1 commented on website/docs/docs/collaborate/legacy-dbt-docs.md:
```diff
@@ -0,0 +1,65 @@
+dbt Docs is a legacy feature that enables you to generate documentation for your project and data platform, rendering it as a website. The documentation is only updated with new information after a fully successful job run, ensuring accuracy and relevance.
+</p>
+
+The default documentation experience in dbt Cloud is [dbt Explorer](/docs/collaborate/explore-projects), available on [Team or Enterprise plans](https://www.getdbt.com/pricing/), which provides a richer and more interactive experience for understanding your project's resources and lineage. This shift makes dbt Docs a legacy documentation feature in dbt Cloud, and is accessible through dbt Explore ...
```
confirm for tech accuracy

dave-connors-3 replied:
yep! looks right

mirnawong1 commented on website/docs/docs/deploy/artifacts.md:
```diff
@@ -4,13 +4,13 @@ id: "artifacts"
 
 ## Create dbt Cloud Artifacts
 
 While running any job can produce artifacts, you should only associate one production job with a given project to produce the project's artifacts. You can designate this connection in the **Project details** page. To access this page, click the gear icon in the upper right, select **Account Settings**, select your project, and click **Edit** in the lower right. Under **Artifacts**, select the job ...
```
```suggestion
dbt Explorer uses the metadata provided by the [Discovery API](/docs/dbt-cloud-apis/discovery-api) to display the details about [the state of your project](/docs/dbt-cloud-apis/project-state). The metadata that's available depends on the [deployment environment](/docs/deploy/deploy-environments) you've designated as production or staging in your dbt Cloud project.

dbt Explorer automatically retrieves the metadata updates after each job run in the production or staging deployment environment so it always has the latest results for your project.

To view a resource and its metadata, you must define the resource in your project and run a job in the production or staging environment. For details on how to set up a job, refer to [set up a documentation job](/docs/collaborate/set-up-doc-job).

<Expandable alt_header="For legacy dbt Docs">
While running any job can produce artifacts, you should only associate one production job with a given project to produce the project's artifacts. You can designate this connection on the **Project details** page. To access this page, click the gear icon in the upper right, select **Account Settings**, select your project, and click **Edit** in the lower right. Under **Artifacts**, select the jobs you want to produce documentation and source freshness artifacts for.

<Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/project-level-artifact-updated.png" width="70%" title="Configuring Artifacts"/>

If you don't see your job listed, you might need to edit the job and select **Run source freshness** and **Generate docs on run**.
</Expandable>
```

mirnawong1 commented on website/docs/docs/deploy/artifacts.md:
```diff
@@ -4,13 +4,13 @@ id: "artifacts"
 While running any job can produce artifacts, you should only associate one production job with a given project to produce the project's artifacts. You can designate this connection in the **Project details** page. To access this page, click the gear icon in the upper right, select **Account Settings**, select your project, and click **Edit** in the lower right. Under **Artifacts**, select the job ...
 
-<Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/project-level-artifact-updated.png" title="Configuring Artifacts"/>
+<Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/project-level-artifact-updated.png" width="70%" title="Configuring Artifacts"/>
```
```suggestion
```

mirnawong1 committed:
Update website/docs/docs/deploy/artifacts.md

mirnawong1 commented on website/docs/docs/deploy/artifacts.md:
```diff
@@ -4,13 +4,12 @@ id: "artifacts"
 
-<Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/project-level-artifact-updated.png" title="Configuring Artifacts"/>
 
 If you don't see your job listed, you might need to edit the job and select **Run source freshness** and **Generate docs on run**.
```
```suggestion
```

mirnawong1 committed:
Update website/docs/docs/deploy/artifacts.md

mirnawong1 committed:
Update website/docs/docs/deploy/artifacts.md

mirnawong1 committed:
Update artifacts.md

mirnawong1 commented on website/docs/docs/deploy/artifacts.md:
```diff
@@ -4,28 +4,35 @@ id: "artifacts"
 ## Create dbt Cloud Artifacts
 
-While running any job can produce artifacts, you should only associate one production job with a given project to produce the project's artifacts. You can designate this connection in the **Project details** page. To access this page, click the gear icon in the upper right, select **Account Settings**, select your project, and click **Edit** in the lower right. Under **Artifacts**, select the job ...
+[dbt Explorer](/docs/collaborate/explore-projects) uses the metadata provided by the [Discovery API](/docs/dbt-cloud-apis/discovery-api) to display the details about [the state of your project](/docs/dbt-cloud-apis/project-state). The metadata that's available depends on the [deployment environment](/docs/deploy/deploy-environments) you've designated as production or staging in your dbt Cloud pro ...
```
confirm for tech accuracy

dave-connors-3 replied:
I might change that last sentence to something like "dbt Explorer uses metadata from your staging and production environments." -- just don't love the "depends on" phrasing here . 

also, dev metadata 🔜

mirnawong1 commented on website/docs/docs/deploy/artifacts.md:
```diff
@@ -4,28 +4,35 @@ id: "artifacts"
-Note that both the job's commands and the docs generate step (triggered by the **Generate docs on run** checkbox) must succeed during the job invocation for the project-level documentation to be populated or updated.
+### Documentation
 
+When set up, dbt Cloud updates the **Explore** link in the navigation so it links to documentation for this job. This link always directs you to [dbt Explorer](/docs/collaborate/explore-projects), where you can view your project's resources and lineage to gain a better understanding of its latest production state.
```
confirm for tech accuracy

dave-connors-3 replied:
`When set up, dbt Cloud updates the **Explore** link in the navigation so it links to documentation for this job.` i would not say this! explorer is always up to date across all the jobs in the environment, so it's not really "documentation for this job" anymore the way legacy docs was.

mirnawong1 replied:
ok got it! how about:

> Navigate to [dbt Explorer](/docs/collaborate/explore-projects) through the **Explore** link to view your project's resources and lineage to gain a better understanding of its latest production state.

mirnawong1 commented on website/sidebars.js:
```diff
@@ -468,10 +469,11 @@ const sidebarSettings = {
         {
           type: "category",
-          label: "Explore dbt projects",
+          label: "Navigate dbt Explorer",
```
confirm with cameron and azzam if that reads ok?

mirnawong1 replied:
@cafzal how do you feel about adding explorer to the sidebar of the docs: 'navgiate dbt explorer'

mirnawong1 commented on website/snippets/_docs-setup-doc-job.md:
```diff
@@ -0,0 +1,22 @@
+
```
confirm for tech accuracy for explorer

mirnawong1 commented on website/vercel.json:
```diff
@@ -2,6 +2,16 @@
   "trailingSlash": false,
   "redirects": [
+    {
+      "source": "/docs/collaborate/documentation",
```
added redirects

mirnawong1 commented on website/docs/docs/deploy/job-commands.md:
```diff
@@ -33,9 +33,9 @@ Every job invocation automatically includes the [`dbt deps`](/reference/commands
+For every job, you have the option to select the [Generate docs on run](/docs/collaborate/set-up-doc-job) or [Run source freshness](/docs/deploy/source-freshness) checkboxes, enabling you to run the commands automatically. 
 
-**Job outcome Generate docs on run checkbox** &mdash; dbt Cloud executes the `dbt docs generate` command, _after_ the listed commands. If that particular run step in your job fails, the job can still succeed if all subsequent run steps are successful. Read [Build and view your docs](/docs/collaborate/build-and-view-your-docs) for more info.
+**Job outcome Generate docs on run checkbox** &mdash; dbt Cloud executes the `dbt docs generate` command, _after_ the listed commands. If that particular run step in your job fails, the job can still succeed if all subsequent run steps are successful. Read [Build and view your docs](/docs/collaborate/set-up-doc-job) for more info.
```
```suggestion
**Job outcome Generate docs on run checkbox** &mdash; dbt Cloud executes the `dbt docs generate` command, _after_ the listed commands. If that particular run step in your job fails, the job can still succeed if all subsequent run steps are successful. Read [Set up documentation job](/docs/collaborate/set-up-doc-job) for more info.
```

mirnawong1 committed:
Update website/docs/docs/deploy/job-commands.md

mirnawong1 committed:
Update artifacts.md

mirnawong1 commented on website/docs/docs/collaborate/set-up-doc-job.md:
```diff
@@ -0,0 +1,11 @@
+description: "Learn how to set up a documentation job and generate metadata to power dbt Explorer and display details about the state of your dbt project."
+---
+
+You can set up documentation for a job in dbt Cloud when you edit your job settings or create a new job. You need to configure the job to generate docs when it runs, then link that job to your project. This step is necessary to view column and statistics for models, sources, and snapshots in dbt Explorer.
```
link to section

mirnawong1 commented on website/docs/docs/build/documentation.md:
```diff
@@ -19,18 +21,14 @@ pagination_prev: null
 ## Overview
 
-Good documentation for your dbt models will help downstream consumers discover and understand the datasets which you curate for them.
+The default documentation experience in dbt Cloud is [dbt Explorer](/docs/collaborate/explore-projects), available on [Team or Enterprise plans](https://www.getdbt.com/pricing/), which provides a richer and more interactive experience for understanding your project's resources and lineage. dbt Cloud developer and dbt Core users can use [dbt Docs](/docs/collaborate/legacy-dbt-docs), a legacy featu ...
```
added explorer content

mirnawong1 commented on website/docs/docs/deploy/artifacts.md:
```diff
@@ -4,28 +4,35 @@ id: "artifacts"
 ## Create dbt Cloud Artifacts
 
-While running any job can produce artifacts, you should only associate one production job with a given project to produce the project's artifacts. You can designate this connection in the **Project details** page. To access this page, click the gear icon in the upper right, select **Account Settings**, select your project, and click **Edit** in the lower right. Under **Artifacts**, select the job ...
+[dbt Explorer](/docs/collaborate/explore-projects) uses the metadata provided by the [Discovery API](/docs/dbt-cloud-apis/discovery-api) to display the details about [the state of your project](/docs/dbt-cloud-apis/project-state). The metadata that's available depends on the [deployment environment](/docs/deploy/deploy-environments) you've designated as production or staging in your dbt Cloud pro ...
```
```suggestion
[dbt Explorer](/docs/collaborate/explore-projects) uses the metadata provided by the [Discovery API](/docs/dbt-cloud-apis/discovery-api) to display the details about [the state of your project](/docs/dbt-cloud-apis/project-state). dbt Explorer uses metadata from your staging and production [deployment environments](/docs/deploy/deploy-environments) (development environment metadata is coming soon).
```

mirnawong1 committed:
Update website/docs/docs/deploy/artifacts.md

mirnawong1 commented on website/docs/docs/deploy/artifacts.md:
```diff
@@ -4,28 +4,35 @@ id: "artifacts"
-Note that both the job's commands and the docs generate step (triggered by the **Generate docs on run** checkbox) must succeed during the job invocation for the project-level documentation to be populated or updated.
+### Documentation
 
+When set up, dbt Cloud updates the **Explore** link in the navigation so it links to documentation for this job. This link always directs you to [dbt Explorer](/docs/collaborate/explore-projects), where you can view your project's resources and lineage to gain a better understanding of its latest production state.
```
```suggestion
Navigate to [dbt Explorer](/docs/collaborate/explore-projects) through the **Explore** link to view your project's resources and lineage to gain a better understanding of its latest production state. 
```

mirnawong1 commented on website/docs/docs/deploy/artifacts.md:
```diff
@@ -4,28 +4,35 @@ id: "artifacts"
 ## Create dbt Cloud Artifacts
 
-While running any job can produce artifacts, you should only associate one production job with a given project to produce the project's artifacts. You can designate this connection in the **Project details** page. To access this page, click the gear icon in the upper right, select **Account Settings**, select your project, and click **Edit** in the lower right. Under **Artifacts**, select the job ...
+[dbt Explorer](/docs/collaborate/explore-projects) uses the metadata provided by the [Discovery API](/docs/dbt-cloud-apis/discovery-api) to display the details about [the state of your project](/docs/dbt-cloud-apis/project-state). dbt Explorer uses metadata from your staging and production [deployment environments](/docs/deploy/deploy-environments) (development environment metadata is coming soon ...
```
reworded and added a 'dev metadata coming soon' item

mirnawong1 committed:
Update website/docs/docs/deploy/artifacts.md

dave-connors-3 commented on website/docs/docs/collaborate/legacy-dbt-docs.md:
```diff
@@ -0,0 +1,65 @@
+
+The default documentation experience in dbt Cloud is [dbt Explorer](/docs/collaborate/explore-projects), available on [Team or Enterprise plans](https://www.getdbt.com/pricing/), which provides a richer and more interactive experience for understanding your project's resources and lineage. This shift makes dbt Docs a legacy documentation feature in dbt Cloud, and is accessible through dbt Explore ...
+
+This document guides you on how to use dbt Docs to generate a website from your dbt project using the `dbt docs generate` command. It provides a central location to view your project's resources, such as models, tests, and metrics, and lineage  &mdash; and helps you understand the data in your warehouse. dbt Docs is currently available on developer plans or dbt Core users.
```
```suggestion
This document guides you on how to use dbt Docs to generate a static website from your dbt project using the `dbt docs generate` command. It provides a point-in-time view of your project's resources, such as models, tests, and metrics, and lineage  &mdash; and helps you understand the data in your warehouse. dbt Docs is currently available on developer plans or dbt Core users.
```

dave-connors-3 commented on website/docs/docs/collaborate/legacy-dbt-docs.md:
```diff
@@ -0,0 +1,65 @@
+
+Refer to [Documentation](/docs/build/documentation) for more configuration details.
+
+## Set up a documentation job
```
I think we can kill this section since it's duplicative of the new page

dave-connors-3 commented on website/docs/docs/collaborate/legacy-dbt-docs.md:
```diff
@@ -0,0 +1,65 @@
+Once you set up a job to generate [documentation](/docs/build/documentation) for your project, you can view documentation in the following ways:
+
+- In the dbt Cloud IDE, click the icon above the file tree in the editor to see the latest version of your documentation rendered in a new browser window. View your documentation for your dbt project while your code is still in development. With this workflow, you can inspect and verify what your project's generated documentation will look like before your changes are released to production.
+- In the dbt Cloud user interface, click **Documentation** in the navigation to view your project's documentation. This link will always help you find the most recent version of your project's documentation in dbt Cloud.
```
This does not exist anymore! we should remove this bullet. docs are still available on the job run page i believe

mirnawong1 replied:
oh yea that's right, thank you `Dave!

mirnawong1 committed:
Update website/docs/docs/collaborate/legacy-dbt-docs.md

Co-authored-by: dave-connors-3 <73915542+dave-connors-3@users.noreply.github.com>

Commit:
add

Commit:
Merge branch 'mwong-update-build-docs' of https://github.com/dbt-labs/docs.getdbt.com into mwong-update-build-docs

Commit:
katherine's feedback

mirnawong1 committed:
Merge branch 'current' into mwong-update-build-docs

Commit:
add

cafzal commented on website/docs/docs/collaborate/explore-projects.md:
```diff
@@ -1,28 +1,33 @@
-To view a resource and its metadata, you must define the resource in your project and run a job in the production or staging environment. The resulting metadata depends on the [commands](/docs/deploy/job-commands) executed by the jobs. 
+- dbt Explorer automatically retrieves the metadata updates after each job run in the production or staging deployment environment so it always has the latest results for your project. 
+- Note that CI jobs do not update dbt Explorer. This is because they don't reflect the production state and don't provide the necessary metadata updates.
+- To view a resource and its metadata, you must define the resource in your project and run a job in the production or staging environment. For details on how to set up a job, refer to [set up a documentation job](/docs/collaborate/set-up-doc-job).
```
@mirnawong1 please drop reference to documentation job. that's more relevant to docs. reader can just follow the next section about generating metadata.

cafzal commented on website/docs/docs/build/documentation.md:
```diff
@@ -60,13 +58,14 @@ models:
-
 ## Generating project documentation
-You can generate a documentation site for your project (with or without descriptions) using the CLI.
+You can generate a documentation site for your project (with or without descriptions) in development using your editor or with [dbt Explorer](/docs/collaborate/explore-projects) in dbt Cloud after a successful job run in the deployment environment.
```
@mirnawong1 i'd frame this as 'you can use explorer to view the latest metadata about your project, or generate a static docs site'. then point the reader to the explorer docs to read about setup (within the generate metadata section). no need to link to the docs job article for explorer setup. generating docs is still an option for folks who don't have explorer or want more flexibility, but shouldn't be our default suggestion.

cafzal commented on website/docs/docs/deploy/artifacts.md:
```diff
@@ -4,28 +4,35 @@ id: "artifacts"
 ## Create dbt Cloud Artifacts
 
-While running any job can produce artifacts, you should only associate one production job with a given project to produce the project's artifacts. You can designate this connection in the **Project details** page. To access this page, click the gear icon in the upper right, select **Account Settings**, select your project, and click **Edit** in the lower right. Under **Artifacts**, select the job ...
+[dbt Explorer](/docs/collaborate/explore-projects) uses the metadata provided by the [Discovery API](/docs/dbt-cloud-apis/discovery-api) to display the details about [the state of your project](/docs/dbt-cloud-apis/project-state). dbt Explorer uses metadata from your staging and production [deployment environments](/docs/deploy/deploy-environments) (development environment metadata is coming soon ...
```
this is a bit confusing. how do we reconcile with the explorer setup section? https://docs.getdbt.com/docs/collaborate/explore-projects#generate-metadata

explorer users don't need to specify which jobs generate the artifacts, just need to ensure the commands are running within some jobs in their environment

mirnawong1 replied:
ok i've used the same content in the 'generate metadata' section here and removed the 'set up docs job' reference:

how does this look @cafzal :
> [dbt Explorer](/docs/collaborate/explore-projects#generate-metadata) uses the metadata provided by the [Discovery API](/docs/dbt-cloud-apis/discovery-api) to display the details about [the state of your project](/docs/dbt-cloud-apis/project-state). It uses metadata from your staging and production [deployment environments](/docs/deploy/deploy-environments) (development environment metadata is coming soon).

> dbt Explorer automatically retrieves the metadata updates after each job run in the production or staging deployment environment so it always has the latest results for your project &mdash; meaning it's always automatically updated after each job run.

> To view a resource and its metadata, you must define the resource in your project and run a job in the production or staging environment.

cafzal commented on website/docs/docs/deploy/artifacts.md:
```diff
@@ -4,28 +4,35 @@ id: "artifacts"
+
+<Expandable alt_header="For legacy dbt Docs">
+
+While running any job can produce artifacts, you should only associate one production job with a given project to produce the project's artifacts. You can designate this connection on the **Project details** page. To access this page, click the gear icon in the upper right, select **Account Settings**, select your project, and click **Edit** in the lower right. Under **Artifacts**, select the job ...
```
good clarification - specifying an artifacts job is only needed for the legacy docs, not explorer

cafzal commented on website/docs/docs/deploy/artifacts.md:
```diff
@@ -4,28 +4,35 @@ id: "artifacts"
+Navigate to [dbt Explorer](/docs/collaborate/explore-projects) through the **Explore** link to view your project's resources and lineage to gain a better understanding of its latest production state. 
 
-<Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/doc-menu.png" title="A link to the latest documentation for the selected job"/>
+Note that both the job's commands and the docs generate step ([triggered](/docs/collaborate/set-up-doc-job) by the **Generate docs on run** checkbox) must succeed during the job invocation for the project-level documentation to be populated or updated.
```
explorer doesn't strictly require docs generate, and the command can also be run manually within a job run step, not just the checkbox. we can just direct folks to https://docs.getdbt.com/docs/collaborate/explore-projects#generate-metadata

cafzal commented on website/docs/docs/deploy/artifacts.md:
```diff
@@ -4,28 +4,35 @@ id: "artifacts"
-<Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/doc-menu.png" title="A link to the latest documentation for the selected job"/>
+Note that both the job's commands and the docs generate step ([triggered](/docs/collaborate/set-up-doc-job) by the **Generate docs on run** checkbox) must succeed during the job invocation for the project-level documentation to be populated or updated.
 
 ### Source Freshness
```
ahh this section/feature is confusing. can we just direct folks to the sources page in explorer by default? like the legacy docs, we're no longer investing in the legacy freshness dashboard

cafzal reviewed (changes requested):
great progress! flagged some edits, happy to discuss further

Commit:
Merge branch 'mwong-update-build-docs' of https://github.com/dbt-labs/docs.getdbt.com into mwong-update-build-docs

Commit:
cameron's feedback

mirnawong1 committed:
Merge branch 'current' into mwong-update-build-docs

Commit:
typo

Commit:
add

mirnawong1 committed:
Merge branch 'current' into mwong-update-build-docs

Commit:
remove legacy dbt docs file and readd build your docs

Commit:
update links

Commit:
update doc

Commit:
rework

Commit:
fix links

mirnawong1 commented on website/docs/docs/build/documentation.md:
```diff
@@ -60,13 +56,19 @@ models:
-You can generate a documentation site for your project (with or without descriptions) using the CLI.
 
-First, run `dbt docs generate` — this command tells dbt to compile relevant information about your dbt project and warehouse into `manifest.json` and `catalog.json` files respectively. To see the documentation for all columns and not just columns described in your project, ensure that you have created the models with `dbt run` beforehand.
+The default documentation experience in dbt Cloud is [dbt Explorer](/docs/collaborate/explore-projects), available on [Team or Enterprise plans](https://www.getdbt.com/pricing/). Use dbt Explorer to view your project's resources (such as models, tests, and metrics), its [metadata](/docs/collaborate/explore-projects#generate-metadata), and their lineage to gain a better understanding of its latest ...
```
@cafzal introducing explorer content here - what do you think?

also moved this page from collaborate -> build your dag. however, there is also an option to have a 'doc your project' folder under the 'build' section. and add doc examples for other resources per Dave's suggestion.

cafzal replied:
sounds good!

mirnawong1 commented on website/docs/docs/build/documentation.md:
```diff
@@ -128,9 +129,11 @@ models:
 
-The "overview" shown in the documentation website can be overridden by supplying your own docs block called `__overview__`. By default, dbt supplies an overview with helpful information about the docs site itself. Depending on your needs, it may be a good idea to override this docs block with specific information about your company style guide, links to reports, or information about who to contac ...
+### Setting a custom overview
+*Currently available for the legacy dbt Docs only.*
```
adding this clarification. opted for this small italics rather than separating this with a header since e some of these configs will be available in explorer.

mirnawong1 commented on website/docs/docs/build/documentation.md:
```diff
@@ -173,32 +177,47 @@ up to page views and sessions.
-Using the docs interface, you can navigate to the documentation for a specific model. That might look something like this:
+### Navigating the documentation site
+
+Use [dbt Explorer](/docs/collaborate/explore-projects) for a richer documentation experience and more interactive experience for understanding your project's resources and lineage. Available on [Team or Enterprise plans](https://www.getdbt.com/pricing/).
```
@cafzal introducing explorer content here - what do you think?

mirnawong1 commented on website/docs/docs/build/documentation.md:
```diff
@@ -173,32 +177,47 @@ up to page views and sessions.
+
+### Deploying the documentation site
+
+With dbt Cloud, use [dbt Explorer](/docs/collaborate/explore-projects) automatically retrieves the metadata updates after each job run in the production or staging deployment environment so it always has the latest results for your project &mdash; meaning it's always automatically updated after each job run.
```
@cafzal introducing explorer content here - what do you think?

mirnawong1 commented on website/docs/docs/collaborate/build-and-view-your-docs.md:
```diff
@@ -0,0 +1,87 @@
+dbt Cloud enables you to generate documentation for your project and data platform. The documentation is automatically updated with new information after a fully successful job run, ensuring accuracy and relevance.
+</p>
+
+The default documentation experience in dbt Cloud is [dbt Explorer](/docs/collaborate/explore-projects), available on [Team or Enterprise plans](https://www.getdbt.com/pricing/). Use [dbt Explorer](/docs/collaborate/explore-projects) to view your project's resources (such as models, tests, and metrics) and their lineage to gain a better understanding of its latest production state.
```
@cafzal reworked this page to add explorer content here as the default experience. then included a dbt docs section lower in the page - what do you think?

mirnawong1 commented on website/docs/docs/collaborate/build-and-view-your-docs.md:
```diff
@@ -0,0 +1,87 @@
+
+:::
+
+## dbt Docs
```
dbt docs section

mirnawong1 commented on website/docs/docs/deploy/artifacts.md:
```diff
@@ -4,31 +4,54 @@ id: "artifacts"
 ## Create dbt Cloud Artifacts
 
-While running any job can produce artifacts, you should only associate one production job with a given project to produce the project's artifacts. You can designate this connection in the **Project details** page. To access this page, click the gear icon in the upper right, select **Account Settings**, select your project, and click **Edit** in the lower right. Under **Artifacts**, select the job ...
+[dbt Explorer](/docs/collaborate/explore-projects#generate-metadata) uses the metadata provided by the [Discovery API](/docs/dbt-cloud-apis/discovery-api) to display the details about [the state of your project](/docs/dbt-cloud-apis/project-state). It uses metadata from your staging and production [deployment environments](/docs/deploy/deploy-environments) (development environment metadata is com ...
```
@cafzal reworked this page to be much simpler and link to the generate metadata page.a

mirnawong1 committed:
Update documentation.md

mirnawong1 commented on website/docs/docs/collaborate/explore-projects.md:
```diff
@@ -1,39 +1,44 @@
 ---
-title: "Explore your dbt projects"
-sidebar_label: "Explore dbt projects"
+title: "Navigate with dbt Explorer"
```
how do we feel about adding explorer in the title?

mirnawong1 replied:
```suggestion
title: "Discovery your projects with dbt Explorer"
```

mirnawong1 commented on website/sidebars.js:
```diff
@@ -468,7 +469,7 @@ const sidebarSettings = {
         {
           type: "category",
-          label: "Explore dbt projects",
+          label: "Navigate dbt Explorer",
```
suggested by katherine
```suggestion
          label: "Discover with dbt Explorer",
```

mirnawong1 committed:
Update website/sidebars.js

mirnawong1 commented on website/sidebars.js:
```diff
@@ -468,7 +469,7 @@ const sidebarSettings = {
         {
           type: "category",
-          label: "Explore dbt projects",
+          label: "Discover with dbt Explorer",
```
```suggestion
          label: "Discover projects with dbt Explorer",
```

mirnawong1 committed:
Update website/sidebars.js

mirnawong1 commented on website/sidebars.js:
```diff
@@ -468,7 +469,7 @@ const sidebarSettings = {
         {
           type: "category",
-          label: "Explore dbt projects",
+          label: "Discover projects with dbt Explorer",
```
@cafzal how do we feel about this new sidebar label to further improve discoverability? suggested by katherine.

mirnawong1 committed:
Update collaborate-with-others.md

mirnawong1 commented on website/docs/docs/build/documentation.md:
```diff
@@ -60,13 +56,19 @@ models:
-First, run `dbt docs generate` — this command tells dbt to compile relevant information about your dbt project and warehouse into `manifest.json` and `catalog.json` files respectively. To see the documentation for all columns and not just columns described in your project, ensure that you have created the models with `dbt run` beforehand.
+The default documentation experience in dbt Cloud is [dbt Explorer](/docs/collaborate/explore-projects), available on [Team or Enterprise plans](https://www.getdbt.com/pricing/). Use dbt Explorer to view your project's resources (such as models, tests, and metrics), its [metadata](/docs/collaborate/explore-projects#generate-metadata), and their lineage to gain a better understanding of its latest ...
+
+dbt Cloud developer and dbt Core users can use [dbt Docs](/docs/collaborate/build-and-view-your-docs#dbt-docs), a legacy feature that generates basic documentation, but it doesn't offer the same speed, metadata, or visibility as dbt Explorer. dbt Docs is available to dbt Cloud developer plans or dbt Core users.
```
```suggestion
dbt Cloud developer and dbt Core users can use [dbt Docs](/docs/collaborate/build-and-view-your-docs#dbt-docs), a legacy feature that generates basic documentation, but it doesn't offer the same speed, metadata, or visibility as dbt Explorer. 
```

mirnawong1 committed:
Update website/docs/docs/build/documentation.md

mirnawong1 committed:
Update website/docs/docs/collaborate/explore-projects.md

mirnawong1 commented on website/docs/docs/collaborate/explore-projects.md:
```diff
@@ -1,39 +1,44 @@
-title: "Explore your dbt projects"
-sidebar_label: "Explore dbt projects"
+title: "Discovery your projects with dbt Explorer"
+sidebar_label: "Navigate with dbt Explorer"
```
```suggestion
sidebar_label: "Discovery your projects with dbt Explorer"
```

mirnawong1 committed:
Update website/docs/docs/collaborate/explore-projects.md

Commit:
update titles

cafzal commented on website/docs/docs/collaborate/explore-projects.md:
```diff
@@ -1,39 +1,44 @@
 ---
-title: "Explore your dbt projects"
-sidebar_label: "Explore dbt projects"
+title: "Discover projects with dbt Explorer"
```
"Discover data" makes more sense

cafzal commented on website/docs/docs/collaborate/explore-projects.md:
```diff
@@ -1,39 +1,44 @@
-title: "Explore your dbt projects"
-sidebar_label: "Explore dbt projects"
+title: "Discover projects with dbt Explorer"
+sidebar_label: "Discover projects with dbt Explorer"
```
"Discover data with dbt Explorer" ?

cafzal commented on website/docs/docs/collaborate/explore-projects.md:
```diff
@@ -1,39 +1,44 @@
-sidebar_label: "Explore dbt projects"
+title: "Discover projects with dbt Explorer"
+sidebar_label: "Discover projects with dbt Explorer"
 description: "Learn about dbt Explorer and how to interact with it to understand, improve, and leverage your data pipelines."
```
"...understand, improve, and leverage your dbt projects"

cafzal reviewed (changes requested):
coming along - sugg minor edits. the full lineage graph example showed the nav bar in the preview, but maybe that was a quirk

Commit:
update to discover data

mirnawong1 committed:
Merge branch 'current' into mwong-update-build-docs

mirnawong1 committed:
Update documentation.md

mirnawong1 committed:
Update build-and-view-your-docs.md

mirnawong1 committed:
Update explore-projects.md

matthewshaver commented on website/docs/docs/build/documentation.md:
```diff
@@ -1,11 +1,12 @@
-pagination_prev: null
 ---
 
+Good documentation for your dbt models will help downstream consumers discover and understand the datasets which you curate for them.
```
```suggestion
Good documentation for your dbt models will help downstream consumers discover and understand the datasets you curate for them.
```

matthewshaver commented on website/docs/docs/build/documentation.md:
```diff
@@ -1,11 +1,12 @@
+Good documentation for your dbt models will help downstream consumers discover and understand the datasets which you curate for them.
+dbt provides a way to generate documentation for your dbt project and render it as a website.
+
 ## Related documentation
```
Should we move this to the bottom while we're here, since that's where we usually place it?

mirnawong1 replied:
tbh i did think that but the other 'build' docs have this format so left it so it would be consistent for users.

matthewshaver commented on website/docs/docs/build/documentation.md:
```diff
@@ -60,13 +55,19 @@ models:
+
+Generate documentation for you project by following these steps:
+
+- First, run `dbt docs generate` — this command tells dbt to compile relevant information about your dbt project and warehouse into `manifest.json` and `catalog.json` files respectively. 
```
Since we're giving them steps to follow in order, suggest a numbered list
```suggestion
1. Run `dbt docs generate` — this command tells dbt to compile relevant information about your dbt project and warehouse into `manifest.json` and `catalog.json` files, respectively. 
```

matthewshaver commented on website/docs/docs/build/documentation.md:
```diff
@@ -60,13 +55,19 @@ models:
+Generate documentation for you project by following these steps:
+
+- First, run `dbt docs generate` — this command tells dbt to compile relevant information about your dbt project and warehouse into `manifest.json` and `catalog.json` files respectively. 
+- To view the documentation for all columns and not just columns described in your project, ensure that you have created the models with `dbt run` beforehand.
```
```suggestion
2. Ensure that you have created the models with `dbt run` to view the documentation for all columns, not just those described in your project.
```

matthewshaver commented on website/docs/docs/build/documentation.md:
```diff
@@ -60,13 +55,19 @@ models:
+
+- First, run `dbt docs generate` — this command tells dbt to compile relevant information about your dbt project and warehouse into `manifest.json` and `catalog.json` files respectively. 
+- To view the documentation for all columns and not just columns described in your project, ensure that you have created the models with `dbt run` beforehand.
+- If you're developing locally, run `dbt docs serve` to use these `.json` files to populate a local website.
```
```suggestion
3. Run `dbt docs serve` if you're developing locally to use these `.json` files to populate a local website.
```

matthewshaver commented on website/docs/docs/build/documentation.md:
```diff
@@ -128,9 +128,11 @@ models:
 
+
 ## Setting a custom overview
+*Currently available for the legacy dbt Docs only.*
```
Should we be referring to it as `legacy` since it's still the docs experience for Core users? They'll be reading this page too, right? Maybe the "Previous dbt Cloud docs experience" or something similar?

matthewshaver commented on website/docs/docs/build/documentation.md:
```diff
@@ -174,31 +177,46 @@ up to page views and sessions.
 dbt's documentation website was built to make it easy to host on the web. The site is "static,” meaning you don't need any "dynamic" servers to serve the docs. You can host your documentation in several ways:
 
-* Use [dbt Cloud](/docs/collaborate/documentation)
+* Use [dbt Cloud](/docs/collaborate/build-and-view-your-docs)'s default documentation experience with [dbt Explorer](/docs/collaborate/explore-projects).
```
```suggestion
* Use [dbt Cloud's](/docs/collaborate/build-and-view-your-docs) default documentation experience with [dbt Explorer](/docs/collaborate/explore-projects).
```

matthewshaver commented on website/docs/docs/collaborate/build-and-view-your-docs.md:
```diff
@@ -0,0 +1,85 @@
+
+## dbt Docs
+
+dbt Docs, available on developer plans or dbt Core users, is a legacy feature that generates a website from your dbt project using the `dbt docs generate` command. It provides a central location to view your project's resources, such as models, tests, and metrics, and lineage  &mdash; and helps you understand the data in your warehouse.
```
Same feedback as above, referring to it as a `legacy` feature when it's still a Core feature

mirnawong1 committed:
Update website/docs/docs/build/documentation.md

Co-authored-by: Matt Shaver <60105315+matthewshaver@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/docs/build/documentation.md

Co-authored-by: Matt Shaver <60105315+matthewshaver@users.noreply.github.com>

Commit:
update

mirnawong1 committed:
Update website/docs/docs/build/documentation.md

Co-authored-by: Matt Shaver <60105315+matthewshaver@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/docs/build/documentation.md

Co-authored-by: Matt Shaver <60105315+matthewshaver@users.noreply.github.com>

Commit:
updates

Commit:
Merge branch 'mwong-update-build-docs' of https://github.com/dbt-labs/docs.getdbt.com into mwong-update-build-docs

mirnawong1 committed:
Update website/docs/docs/build/documentation.md

Co-authored-by: Matt Shaver <60105315+matthewshaver@users.noreply.github.com>

mirnawong1 committed:
Merge branch 'current' into mwong-update-build-docs

mirnawong1 commented on website/vercel.json:
```diff
@@ -2,6 +2,11 @@
   "cleanUrls": true,
   "trailingSlash": false,
   "redirects": [
+    {
```
added this just in case

Commit:
adding redirect

mirnawong1 committed:
Merge branch 'current' into mwong-update-build-docs

mirnawong1 committed:
Update sharing-documentation.md (#5644)

this pr updates the doc sharing faq to provide users the available
options to view documentation (explorer and dbt docs)

Commit:
update

Commit:
add plan
